debian/paasta-tools.links

```diff
@@ -48,4 +48,5 @@ opt/venvs/paasta-tools/bin/paasta_setup_chronos_job usr/bin/setup_chronos_job
 opt/venvs/paasta-tools/bin/paasta_tabcomplete.sh etc/bash_completion.d/paasta.bash
 opt/venvs/paasta-tools/bin/paasta usr/bin/paasta
 opt/venvs/paasta-tools/bin/setup_marathon_job.py usr/bin/setup_marathon_job
+opt/venvs/paasta-tools/bin/update_soa_memcpu.py usr/bin/update_soa_memcpu
```

Comment:
can you put paasta_ in the name somewhere to make it not clutter the system path. Do it in setup.py as well


Also please keep this sorted

Reply:
Since there is already this line: https://github.com/Yelp/paasta/blob/master/setup.py#L103

Do I still need to add a line to `console_scripts` ?